rollingpin/deploy.py

```diff
@@ -158,8 +161,9 @@ def signal_handler(sig, _):
                     for build_hostname, build_refs in by_buildhost.iteritems():
                         build_command = ["build"] + build_refs
                         build_host = Host.from_hostname(build_hostname)
-                        (tokens,) = yield self.process_host(
+                        (results,) = yield self.process_host(
```

Comment:
Ah, I am mistaken. Seems like we're returning a tuple of a tuple from `process_host` and you ignore the second item and unpack the results later on Line 138